aws_quickstart/datadog_agentless_scanning.yaml

```diff
@@ -41,6 +41,15 @@ Parameters:
       Enable Agentless Scanning of Lambda vulnerabilities. "CloudSecurityPostureManagement" must be set to 'true'.
     Default: false
 
+  AgentlessSensitiveDataScanning:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+    Description: >-
+      Enable Agentless Scanning of datastores (S3 buckets, RDS databases). "CloudSecurityPostureManagement" must be set to 'true'.
```

Comment:
I think it might be better to have separate flags for S3 and RDS scanning (in case you want to enable one but not the other).